homeassistant/components/calendar/google.py

```diff
@@ -89,7 +89,7 @@ def _prepare_query(self):
         params['timeMax'] = end_date.isoformat('T')
 
         events = await hass.async_add_job(service.events)
-        result = await hass.async_add_job(events.list(**params).execute)
+        result = await hass.async_create_task(events.list(**params).execute)
```

Comment:
async_create_task can only be used with coroutines. This looks like it's a sync method.

And actually, this whole method should just be 1 method that is executed in the executor (async_add_executor_job) and not call to executor 3 times, that's never faster. (I know, unrelated to this work)